l2-contracts/hardhat.config.ts

```diff
@@ -37,6 +45,15 @@ export default {
   solidity: {
     version: "0.8.20",
   },
+  solpp: {
```

Comment:
why is this needed?

Reply:
Thought, that it passes some params to deployer. Now it's removed.